src/auth/hooks/useAuthProvider.ts

```diff
@@ -65,7 +70,12 @@ export function useAuthProvider({
   }, [authenticated]);
 
   useEffect(() => {
-    if (!authenticated && !authenticating && permitCredentialsAPI.current) {
+    if (
+      !authenticated &&
+      !authenticating &&
+      !requestedExternalPluginId &&
```

Comment:
```suggestion
      !requestedExternalPluginId &&
```
doesn't this condition always resolve to `true` when coupled with:
```
    setRequestedExternalPluginId(null);
```
on sucess external authentication? Because the initial value from localStorage will be null and the value you get from `useLocalStorage` is not reactive afaik (it does not listen on changes).

Reply:
Hmm this logic makes sense between redirects, so if that was your intention it seems 👍

Reply:
Yes, it should work between redirects

Reply:
I guess `requestedExternalPluginId` it's used by sdk?

Reply:
It is used by `LoginView`

Reply:
^ To determine what external plugin was used to log in, after redirect